### PR TITLE
Update Visual Studio installation steps to current

### PR DESCRIPTION
**Pull request by jwrober**
No related issue.

### CODE REVIEW

File: docs/Contributing/visual-studio.rst [hunk 1]
```diff
@@ -16,41 +16,43 @@ Base Installation
...
+Double-click the :file:`VisualStudioSetup.exe` file in your Downloads folder and follow the prompts to get the
+setup process started. At some point you will be prompted to select the type of workload you want to
+install. Select :guilabel:`Python Development` and :guilabel:`Desktop development with C++`. On the right
+panel, under :guilabel:`Installation Details`, uncheck :guilabel:`vcpkg package manager` and check
+:guilabel:`Windows 10 SDK 10.0.20348.0)`.
+
+.. note:: We remove the ``vcpkg`` package manager inside of Visual Studio in order to install one that is
+   better managed and easily updated later down this page.
+
+Next click on the Individual components tab and select the following options: :guilabel:`Git for Windows`,
```
Comment (blabber) [inline docs/Contributing/visual-studio.rst:28]: :guilabel:`Individual components` ?

**Review by blabber — APPROVED** [review]
Apart from the longstanding formatting nitpick this looks good.

**Comment by jwrober** [comment]
> Apart from the longstanding formatting nitpick this looks good.

Formatting nitpick?

**Comment by blabber** [comment]
I meant the question if "Individual components" should be tagged as `guilabel`. Not being a native speaker I probably did not express myself too well 😅

What I meant: 1. It's not a big issue (nitpick) and 2. it's been this way a while (longstanding). I approved the PR because I am not sure if this really is an issue and your changes look good :)

**Comment by jwrober** [comment]
Sounds good! Fun w/ Engrish LoL

**Pull request merged**